apps/web/src/server/applications/case/s51/applications-s51.controller.js

```diff
@@ -176,14 +181,16 @@ export async function changeAdviceStatus(request, response) {
 	 * @type {{ id: number; }[]}
 	 */
 
-	const items = body.selectedFilesIds.map((/** @type {any} */ selectField) => ({ id: Number(selectField) }));
+	const items = body.selectedFilesIds.map((/** @type {any} */ selectField) => ({
+		id: Number(selectField)
+	}));
 
 	const payload = {
 		// @ts-ignore
-		redacted: body.isRedacted === "undefind" ? body.isRedacted : body.isRedacted === '1',
+		redacted: body.isRedacted === 'undefind' ? body.isRedacted : body.isRedacted === '1',
```

Comment:
should this be 'undefined' ?

Reply:
I very much think so. 
It's out-of-scope because the change was just on the quotes, but I'll fix it